src/tabs/Tabs.styles.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+
+const TabsStyles = styled.nav`
+  display: flex;
+  align-items: stretch;
+  column-gap: 4px;
+  padding: 0 2.5px;
+  justify-content: flex-start;
+  border-bottom: 1px solid ${({ theme }) => theme.c7__ui.borderColor};
+  margin: 0 -20px;
```

Comment:
I think we should build this without the negative margin - we need it in admin but if another app uses this its going to get out of the layout. Maybe we should just do margin 0 and override it in admin? For this and the TabBody below

Looks good other than this!